integration-libs/cdc/root/config/default-cdc-routing-config.ts

```diff
@@ -22,6 +22,16 @@ export const cdcRoutesConfig: RoutesConfig = {
     protected: false,
     authFlow: true,
   },
+  login: {
+    paths: ['login'],
+    protected: false,
+    authFlow: true,
+  },
+  loginForm: {
+    paths: ['login-form'], //overriding to avoid same path in different routes
```

Comment:
fine with override `loginForm`, but I think if you remove paths from here, as it's unnecessary
```suggestion
```

Reply:
or 
```suggestion
    paths: [], //overriding to avoid same path in different routes
```
If issue will still raise

Reply:
done